sdk/communication/communication-rooms/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.1.0-beta.1 (2023-09-21)
+
+### Features Added
+
+- Added Support for PSTN DialOut Capability in Azure Communication Services Rooms
```

Comment:
write this with normal sentence capitalization (there is also no need to repeat ACS rooms since it's pretty obvious): 
- `Added support for PSTN dialout capability`

Reply:
Ok let me update. thanks.